File: fixtures/csharp/baseline/src/Service.cs
namespace Example;

public static class Defaults
{
    public const string Prefix = "job";
}

public class Repository
{
    public string Last { get; private set; } = "";

    public string Save(string value)
    {
        Last = value.Trim();
        return Last;
    }
}

public class Service
{
    private readonly Repository repository;

    public Service(Repository repository)
    {
        this.repository = repository;
    }

    public string Execute(string name)
    {
        var stored = repository.Save(name);
        return $"{Defaults.Prefix}:{stored}";
    }
}

File: fixtures/php/baseline/src/Service.php
<?php

namespace Example;

final class Defaults
{
    public const PREFIX = 'job';
}

final class Repository
{
    public string $last = '';

    public function save(string $value): string
    {
        $this->last = trim($value);
        return $this->last;
    }
}

final class Service
{
    public function __construct(private Repository $repository)
    {
    }

    public function execute(string $name): string
    {
        $stored = $this->repository->save($name);
        return Defaults::PREFIX . ':' . $stored;
    }
}
